src/curies/api.py

```diff
@@ -2191,3 +2194,85 @@ def _get_shacl_line(prefix: str, uri_prefix: str, pattern: Optional[str] = None)
         pattern = pattern.replace("\\", "\\\\")
         line += f'; sh:pattern "{pattern}"'
     return line + " ]"
+
+
+def upgrade_prefix_map(prefix_map: Mapping[str, str]) -> List[Record]:
```

Comment:
LGTM! I'm sure your test passes as well, but I ran it locally on my own and worked for me as well.